src/content-layout/__tests__/content-layout.test.tsx

```diff
@@ -13,16 +15,12 @@ function renderContentLayout(props: ContentLayoutProps = {}) {
   return {
     wrapper,
     isOverlapEnabled() {
-      const backgroundElement = wrapper.findByClassName(styles.background)!.getElement();
-      const classes = backgroundElement.getAttribute('class')!;
+      const classes = wrapper.getElement().getAttribute('class')!;
       return !classes.includes(styles['is-overlap-disabled']);
```

Comment:
The previous version is unsafe, because it matches even a substring like `is-overlap-disabled-something-else`.

Should be like this:

```suggestion
      return wrapper.getElement().classList.contains(styles['is-overlap-disabled'])
```

Reply:
Makes sense, done